app/controllers/workflow/pages_controller.rb

```diff
@@ -6,186 +6,186 @@ class Workflow::PagesController < ApplicationController
 
   private
 
-    def set_model
-      @model = Cms::Page
+  def set_model
+    @model = Cms::Page
+  end
+
+  def set_item
+    @item = @model.find(params[:id]).becomes_with_route
+    @item.attributes = fix_params
+    @item.try(:allow_other_user_files)
+  end
+
+  def fix_params
+    { cur_user: @cur_user, cur_site: @cur_site, cur_node: false }
+  end
+
+  def request_approval
+    current_level = @item.workflow_current_level
+    current_workflow_approvers = @item.workflow_approvers_at(current_level)
+    current_workflow_approvers.each do |workflow_approver|
+      args = { f_uid: @cur_user._id, t_uid: workflow_approver[:user_id],
+               site: @cur_site, page: @item,
+               url: params[:url], comment: params[:workflow_comment] }
+      Workflow::Mailer.request_mail(args).deliver_now rescue nil
     end
 
-    def set_item
-      @item = @model.find(params[:id]).becomes_with_route
-      @item.attributes = fix_params
-      @item.try(:allow_other_user_files)
-    end
-
-    def fix_params
-      { cur_user: @cur_user, cur_site: @cur_site, cur_node: false }
-    end
-
-    def request_approval
+    @item.set_workflow_approver_state_to_request
+    @item.update
+  end
+
+  def workflow_alert
```

Comment:
__[rubocop]__
* Perceived complexity for workflow_alert is too high. [15/14]

<!-- Comment by [SideCI](https://sideci.com) -->